common/djangoapps/student/tests/tests.py

```diff
@@ -563,6 +564,72 @@ def _enrollment_with_complete_course(self, enrollment_mode):
         enrollment = CourseEnrollment.enroll(self.user, self.course.id, mode=enrollment_mode)
         return complete_course_mode_info(self.course.id, enrollment)
 
+    def _create_program_config(self, **kwargs):
```

Comment:
You might wanna refactor this (after my PR is merged) since I have add Base class `ProgramsApiBasicTestConfig` in my PR.